shenyu-admin/src/main/resources/mappers/rule-sqlmap.xml

```diff
@@ -201,8 +200,8 @@
     </insert>
 
     <update id="update" parameterType="org.apache.shenyu.admin.model.entity.RuleDO">
-        update rule
-        set date_created = #{dateCreated,jdbcType=TIMESTAMP},
+        UPDATE rule
+        SET date_created = #{dateCreated,jdbcType=TIMESTAMP},
```

Comment:
The keywords are right-aligned and the fields are left-aligned and displayed in multiple lines.